include/class.schedule.php

```diff
@@ -1049,15 +1049,17 @@ static function getFrequencies() {
     }
 
     static function getDays() {
-        static $translated = false;
-        if (!$translated) {
-            foreach (static::$days as $k=>$v)
-                static::$days[$k] = __($v);
-        }
-
         return static::$days;
     }
 
+    static function getTranslatedDays() {
+        static $translated = null;
+        if (!isset($translated))
+            $translated = array_map(function ($d) { return __($d); }, static::getDays());
```

Comment:
Looks good - but I think `static::getDays()` should be `self::getDays()`. It really doesn't matter in this context but it might down the road.

Reply:
done